alerts/monquorum.libsonnet

```diff
@@ -23,15 +23,15 @@
           {
             alert: 'CephMonHighNumberOfLeaderChanges',
             expr: |||
-              rate(ceph_mon_num_elections{%(cephExporterSelector)s}[15m]) > 3
+              rate(ceph_mon_num_elections{%(cephExporterSelector)s}[5m]) * 60 > 0.95
             ||| % $._config,
             'for': $._config.monQuorumLeaderChangesAlertTime,
             labels: {
               severity: 'warning',
             },
             annotations: {
               message: 'Storage Cluster has seen many leader changes recently.',
-              description: 'Ceph Monitor "{{ $labels.job }}": instance {{ $labels.instance }} has seen {{ $value }} leader changes recently.',
+              description: 'Ceph Monitor "{{ $labels.job }}": instance {{ $labels.instance }} has seen {{ $value }} leader changes per minute recently.',
```

Comment:
can be change `per minute recently` to `frequently`. As per minute is a time duration we might not want to fix that value in message.